### PR TITLE
Can we use chartjs-node-canvas instead of chartjs-node?

### PR DESCRIPTION
**Pull request by sisidovski**
Hi!

I tried to update README.md file, but I couldn't install chartjs-node library on the current master branch because of node-gyp error. Maybe similar to https://github.com/vmpowerio/chartjs-node/issues/85, I couldn't fix this installation issue on my local machine (Mac OS 11.3.1, node.js v16.4.2).

I don't know how reasonable it is, but It looks like chartjs-node is not actively maintained anymore. So can we use [chartjs-node-canvas](https://github.com/SeanSobey/ChartjsNodeCanvas) instead?

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -22,8 +22,8 @@
   },
   "license": "MIT",
   "devDependencies": {
-    "chart.js": "^2.8.0",
-    "chartjs-node": "^1.7.1",
+    "chart.js": "^2.9.4",
+    "chartjs-node-canvas": "^3.2.0",
```
Comment (patrickhulce) [inline package.json:26]: the new legend text is extremely hard to read, could we fix the font color there?

**Review by patrickhulce — COMMENTED** [review]
thanks @sisidovski!

just the one note about color contrast but otherwise this is great :)

File: package.json (hunk 1 above)
Comment (sisidovski) [reply package.json:26]: Ah that is correct, let me check.

**Comment by sisidovski** [comment]
The legend is still slightly different, but now it has the better color contrast. WDYT? @patrickhulce

**Review by patrickhulce — APPROVED** [review]
different it fine this is even more legible :)

thanks very much!

**Pull request merged**